map/map_tests/ge0_parser_tests.cpp

```diff
@@ -81,13 +80,12 @@ bool ConvergenceTest(double lat, double lon, double latEps, double lonEps)
     return true;
   return false;
 }
-
-}  // unnamed namespace
+}  // namespace
 
 UNIT_TEST(Base64DecodingWorksForAValidChar)
 {
   Ge0ParserForTest parser;
-  for (size_t i = 0; i < 64; ++i)
+  for (int i = 0; i < 64; ++i)
```

Comment:
uint8_t?

Reply:
That would require changing the signature of the obscure MapsWithMe_Base64Char function. I'd rather not.

Reply:
ok